src/blocks/weather/met_no.rs

```diff
@@ -283,11 +301,51 @@ impl WeatherProvider for Service<'_> {
             })
         };
 
+        let (sunset, sunrise) = match need_sunrise_and_sunset {
+            true => {
+                let current_date = chrono::Local::now().format("%Y-%m-%d").to_string();
```

Comment:
I'm guessing using the default value provided by met.no is probably the best bet instead of trying to set it myself as I did previously:
![image](https://github.com/greshake/i3status-rust/assets/20906912/a933b176-4211-432f-83b7-de875ea7f205)

Reply:
On Mon, Jun 3, 2024 at 9:20 AM @met.no via RT <@met.no> wrote:

> Well, the intended usage is to specify the date using the so named
> parameter. If missing, the API will get the day from localtime on the
> server, but since that is set to UTC the date will sometimes be different
> than in other timezones.
>
> We might look into converting localtime into the timezone specified. I'll
> put it on the ToDo list.
>
> Regards,
>
> Geir
> API developer
>
>
> On 23/05/2024 03:21, Bryan via RT wrote:
>> Ticket URL: https://ticket.met.no/rt/Ticket/Display.html?id=147037
>>
>> Geir,
>> Thanks for your reply!
>> I did initially think that the offset was required based on the documentation,
>> but even with the offset (and no date specified) I see tomorrow's
>> sunrise/sunset time.
>> https://api.met.no/weatherapi/sunrise/3.0/sun?lat=40.43&lon=-74.01&offset=-04:00
>> (hit at 2024-05-22 21:17 UTC-4) returns:
>> ```json
>> {
>>     "copyright": "MET Norway",
>>     "licenseURL": "https://api.met.no/license_data.html",
>>     "type": "Feature",
>>     "geometry": {
>>         "type": "Point",
>>         "coordinates": [
>>             -74,
>>             40.4
>>         ]
>>     },
>>     "when": {
>>         "interval": [
>>             "2024-05-23T04:52:00Z",
>>             "2024-05-24T04:56:00Z"
>>         ]
>>     },
>>     "properties": {
>>         "body": "Sun",
>>         "sunrise": {
>>             "time": "2024-05-23T05:32-04:00",
>>             "azimuth": 61.53
>>         },
>>         "sunset": {
>>             "time": "2024-05-23T20:13-04:00",
>>             "azimuth": 298.63
>>         },
>>         "solarnoon": {
>>             "time": "2024-05-23T12:52-04:00",
>>             "disc_centre_elevation": 70.36,
>>             "visible": true
>>         },
>>         "solarmidnight": {
>>             "time": "2024-05-23T00:52-04:00",
>>             "disc_centre_elevation": -28.93,
>>             "visible": false
>>         }
>>     }
>> }
>> ````
>> On Wed, May 22, 2024 at 9:27 AM @met.no via RT <@met.no]> wrote:
>>
>>> Hello,
>>>
>>> the intended usage is that you send the current time zone offset in the URL:
>>>> https://api.met.no/weatherapi/sunrise/3.0/sun?lat=59.933333&lon=10.716667&offset=-04:00
>
>>> This way you will get timestamps in local time, and the start and end of the
>>> date will be calculated correctly (e.g. "2024-05-21T23:13:00Z" to
>>> "2024-05-22T23:17:00Z" UTC).
>>>
>>> Without the offset parameter we have no way of knowing the correct time
>>> zone, and by implication won't know when the calendar day starts and ends.
>>> Instead the API will use the longitude to calculate the midday point (when
>>> the sun crosses the meridian) and use the +/- 12 hour interval as the
>>> "date". Also, timestamps will be returned as UTC.
>>>
>>> All this is documented here:
>>> https://docs.api.met.no/doc/sunrise/celestial.html
>>> Hope this helps,
>>>
>>> Geir
>>> API developer
>>>
>>>
>>> On 20/05/2024 04:01, Bryan via RT wrote:
>>>
>>>> Hello, I have a question regarding the default date that is used for the
>>>> sunrise api.
>>>> When I hit
>>>>    https://api.met.no/weatherapi/sunrise/3.0/sun?lat=59.933333&lon=10.716667   at
>>>> 23:00 UTC (19:00 NY local/(UTC-4) on 2024-05-18) I got back:
>>>>
>>>> ```json
>>>> {
>>>>     "copyright": "MET Norway",
>>>>     "licenseURL": "  https://api.met.no/license_data.html  ",
>>>>     "type": "Feature",
>>>>     "geometry": {
>>>>         "type": "Point",
>>>>         "coordinates": [
>>>>             10.7,
>>>>             59.9
>>>>         ]
>>>>     },
>>>>     "when": {
>>>>         "interval": [
>>>>             "2024-05-17T23:13:00Z",
>>>>             "2024-05-18T23:17:00Z"
>>>>         ]
>>>>     },
>>>>     "properties": {
>>>>         "body": "Sun",
>>>>         "sunrise": {
>>>>             "time": "2024-05-18T02:32+00:00",
>>>>             "azimuth": 45.97
>>>>         },
>>>>         "sunset": {
>>>>             "time": "2024-05-18T19:56+00:00",
>>>>             "azimuth": 314.46
>>>>         },
>>>>         "solarnoon": {
>>>>             "time": "2024-05-18T11:13+00:00",
>>>>             "disc_centre_elevation": 49.81,
>>>>             "visible": true
>>>>         },
>>>>         "solarmidnight": {
>>>>             "time": "2024-05-17T23:13+00:00",
>>>>             "disc_centre_elevation": -10.5,
>>>>             "visible": false
>>>>         }
>>>>     }
>>>> }
>>>> ```
>>>> However, after midnight UTC (20:05 NY local/(UTC-4) on 2024-05-18) I see the sunset times for the following day:
>>>> ```json
>>>> {
>>>>     "copyright": "MET Norway",
>>>>     "licenseURL": "  https://api.met.no/license_data.html  ",
>>>>     "type": "Feature",
>>>>     "geometry": {
>>>>         "type": "Point",
>>>>         "coordinates": [
>>>>             10.7,
>>>>             59.9
>>>>         ]
>>>>     },
>>>>     "when": {
>>>>         "interval": [
>>>>             "2024-05-18T23:13:00Z",
>>>>             "2024-05-19T23:17:00Z"
>>>>         ]
>>>>     },
>>>>     "properties": {
>>>>         "body": "Sun",
>>>>         "sunrise": {
>>>>             "time": "2024-05-19T02:30+00:00",
>>>>             "azimuth": 45.41
>>>>         },
>>>>         "sunset": {
>>>>             "time": "2024-05-19T19:58+00:00",
>>>>             "azimuth": 315.01
>>>>         },
>>>>         "solarnoon": {
>>>>             "time": "2024-05-19T11:13+00:00",
>>>>             "disc_centre_elevation": 50.03,
>>>>             "visible": true
>>>>         },
>>>>         "solarmidnight": {
>>>>             "time": "2024-05-18T23:13+00:00",
>>>>             "disc_centre_elevation": -10.28,
>>>>             "visible": false
>>>>         }
>>>>     }
>>>> }
>>>> ```
>>>> Is this the intended behavior, to use the current date in UTC, instead of the local current date when no date is specified?
>>>>
>>>> Thanks in advance!
>>>> //Bryan
>>>>
>>>>
>>
>>